### ISSUE TITLE
Clarify projection of bounding box columns

### ISSUE THREAD

**Issue by kylebarron** [opengeospatial/geoparquet#203]
I think the spec could be a bit clearer about this. Do the bbox columns HAVE to be in the same projection as the data?

Thinking about this in the context of [stac-geoparquet](https://github.com/stac-utils/stac-geoparquet)... the [STAC spec](https://github.com/radiantearth/stac-spec) defines bbox to [_always be WGS84_](https://github.com/radiantearth/stac-spec/blob/v1.0.0/item-spec/item-spec.md#bbox), even when the projection of the geometry is _not_ WGS84. There are certainly some use cases where having the bbox always in WGS84 simplifies things (thinking of the browser myself). 

Is this something that should be allowed in the spec? Should bbox columns be allowed to have their own projection?

**Comment by cholmes** [comment]
I agree there should be a strong use case for bbox columns in a different projection.

It does seem like this could be an area where there's experimentation for awhile before we bring it into the spec. Like there's nothing stopping adding an extra `consistent-bbox` column in WGS84, or even only using that (since the core bbox one is optional). If there's compelling cases for it we could add it.

Not sure if we need to be more explicit in the spec that for now we require the bounding box geometry to be the same as the specified crs?

**Comment by cholmes** [comment]
So do we want to say something in the spec, just to be clear that current practice is for bbox columns to be in the same projection as the primary geometry column?

**Comment by jorisvandenbossche** [comment]
Yes, I think it would be good to add a sentence to explicitly state that the values in the bbox column should be interpreted in the projection of the corresponding geometry column.

(note this is not necessarily the _primary_ geometry column, as the covering bbox info is in the _column_ metadata, so in theory you can have multiple geometry columns in a file each with a different CRS and have a bbox column for each of those)
